var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -192,6 +192,11 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
                 "libint@2.6.0:+fortran tune=cp2k-lmax-{0}".format(lmax),
                 when="@7.0: lmax={0}".format(lmax),
             )
+            # AOCC only works with libint@2.6.0
+            depends_on(
+                "libint@=2.6.0+fortran tune=cp2k-lmax-{0}".format(lmax),
+                when="@7.0: lmax={0} %aocc".format(lmax),
+            )
```

Comment:
```suggestion
            # AOCC only works with libint@2.6.0
            depends_on(
                f"libint@=2.6.0+fortran tune=cp2k-lmax-{lmax}",
                when=f"@7.0: lmax={lmax} %aocc",
            )
```
I think it is recommended to use f-strings now.

Reply:
Thanks
changes pushed as suggested